tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -1682,15 +1679,19 @@ describe('IssueList', function () {
         },
       },
     };
-    wrapper = mountWithTheme(<IssueListOverview {...props} />);
+
+    const {routerContext} = initializeOrg();
+    wrapper = mountWithTheme(<IssueListOverview {...props} />, routerContext);
     wrapper.setState({
       groupIds: range(0, 25).map(String),
       queryCount: 500,
       queryMaxCount: 1000,
+      pageLinks: DEFAULT_LINKS_HEADER,
```

Comment:
Without this the pagination will not render (it thinks there are no pages)

Reply:
bah implementation details :(

Reply:
yeah gross. But this whole test is pretty gross like that tbh